app/styles/components/_footer.scss

```diff
@@ -59,12 +82,25 @@ $icon-size: 25px;
 @media (max-width: $Theme-break-mobile) {
   .Footer {
     padding: $Theme-margin-half $Theme-margin-small;
+    display: flex;
+    flex-direction: column;
+  }
+
+
+  .Footer-column {
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+
+    .Logo {
+      margin-left: auto;
+      margin-right: auto;
+    }
```

Comment:
According to SUIT you should not reference components within other components.

Either way, the previous comment about centering should also apply here.